### ISSUE TITLE
Dark/light mode toggle not accessible by assistive technology

### ISSUE THREAD

**Issue by kangabell** [danny-avila/LibreChat#5390]
The dark/light mode button on the lower left corner of the Prompts panel lacks an accessible name and its functions aren't accessible by keyboard.

## URL
https://librechat-librechat.hf.space/d/prompts/new

## Page Area
Prompts Panel

## Issue Description
The screen reader reads this button as "image on, switch" and gives no context as to what the switch does. Pressing and holding the spacebar causes the interface to flash rapidly between the two modes and there is no way to get it to stay on the opposite mode.

[Screen Recording](https://share.zight.com/kpuzdO9J)

## WCAG Criteria
4.1.2 Name, Role, Value - A
2.1.1 Keyboard - A

## Testing Method
Manual - VO/Chrome/MacOS

## Action
- Ensure the html class (.dark or .light) and the aria-checked attribute on the button are updated properly when hitting the Spacebar and/or Enter key on the keyboard.
- Add aria-hidden="true" to the svg element, and add an aria-label to the <button> with a descriptive name like "____ mode" or "Switch to ___ theme".

## Notes
Accessible names missing is an issue identified by [Accessibility issues for screenreader users #5187](https://github.com/danny-avila/LibreChat/issues/5187)

**Comment by kangabell** [comment]
This might be the unlabeled checkbox referenced in #5187:
>In the prompt createion field, every text box is labelled, there's however an unlabelled checkbox which I don't know what it's meant for.
